coreneuron/nrniv/nrn_setup.cpp

```diff
@@ -784,10 +784,24 @@ inline void mech_layout(data_reader& F, T* data, int cnt, int sz, int layout) {
  * allocated with malloc(). This is not the case here, so let's try and fix
  * things up first. */
 
-void nrn_cleanup() {
+void nrn_cleanup(bool clean_ion_global_map) {
```

Comment:
just to understand: is this change because you want to call `nrn_cleanup` and still want to have access to `ion_global_map`?